mhs/inbound/inbound/request/tests/test_handler.py

```diff
@@ -26,37 +55,87 @@ class TestInboundHandler(tornado.testing.AsyncHTTPTestCase):
     message_dir = pathlib.Path(current_dir) / MESSAGES_DIR
 
     def setUp(self):
-        self.callbacks = {}
+        self.state = unittest.mock.MagicMock()
+        self.state.get = unittest.mock.MagicMock(side_effect=state_return_values)
```

Comment:
I think (but double-check this) that this can be simplified to:
```suggestion
        self.state.get.side_effect=state_return_values
```

Reply:
Works as expected